web/packages/teleport/src/Integrations/status/AwsOidc/Tasks/Tasks.tsx

```diff
@@ -55,6 +60,20 @@ export function Tasks() {
     serverSidePagination.fetch();
   }, [integration]);
 
+  // use updated query params to set/unset the task side panel
+  useEffect(() => {
+    if (
+      taskName &&
+      taskName !== '' &&
+      serverSidePagination.fetchedData.agents &&
+      selectedTask === undefined
```

Comment:
```suggestion
      selectedTask === ""
```
i lead you astray. i still want the default for `selectedTask` to be the empty string like you changed, but now we cant check for `=== undefined` and instead need to check for `""`

Reply:
lol, of course